### ISSUE TITLE
bug of react-pdf

### ISSUE THREAD

**Issue by vadim9999** [GroceriStar/pdf-export-component#19]
it's without header
![image](https://user-images.githubusercontent.com/35640573/50920026-fa3ecb00-144c-11e9-80d6-d3dfeca8362b.png)
but when I add header the space between boxes are random like this
![image](https://user-images.githubusercontent.com/35640573/50919911-c663a580-144c-11e9-9782-90b27c9ca812.png)

**Comment by atherdon** [comment]
noted. do it without header right now

**Comment by atherdon** [comment]
right now i want blocks that will be aligned together. i'm pretty sure that this header issue not a big problem,
like: https://user-images.githubusercontent.com/1469198/39208703-6d242a1a-480c-11e8-9180-3b471ddbd688.jpg

**Comment by vadim9999** [comment]
@atherdon should I create 3 columns in a page?

**Comment by vadim9999** [comment]
https://github.com/GroceriStar/react-print-pdf/pull/5
When I change grocery list the problem also exist, need waiting new relese of render-pdf

**Comment by atherdon** [comment]
@vadim9999 it's not important for me - have we 2 or 3 columns, i think in future we'll be able to adjust it via css rules, by adding some flexibility. right now 2 columns is ok.

Ok, let's wait for a new release. pausing this layout.
Btw, can you setup a 4th layout - i want to be able to print a recipe data too.

**Comment by atherdon** [comment]
react-pdf released an updated version of i's module, can you check if this bug still appears?

**Comment by vadim9999** [comment]
ok

**Comment by vadim9999** [comment]
@atherdon This bug is still appears

**Comment by atherdon** [comment]
ok, then we'll create an issue at diego`s repository

**Comment by atherdon** [comment]
@vadim9999 can you re-check if we still have this bug? you can check it at react-print-pdf repo - i clean it up and fix some things lately...

**Comment by vadim9999** [comment]
react-print-pdf doesn't work it's throw me error that can't find lib of sd-wrapper. I tried to build sd-wrapper, 
this command 
```sudo npm run files```
throw me error  
also files2

**Comment by vadim9999** [comment]
also this problem in fake-api

**Comment by vadim9999** [comment]
Ok we can add dist to repository in and change in path in fetch instead of lib use dist

**Comment by atherdon** [comment]
@vadim9999 yeah i'm aware of this problem. we're waiting when our new intern will fix it

**Comment by vadim9999** [comment]
> @vadim9999 can you re-check if we still have this bug? you can check it at react-print-pdf repo - i clean it up and fix some things lately...

Yes, we still have this bug

**Comment by atherdon** [comment]
I have some doubts about it...
Btw, today i figure out that our react-component(pdf) didn't have the latest packages, which is strange... because i upgraded the renderer module... 
maybe we'll have more time and resources to fix it later
